presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/MetastoreApiPartitionWithStatistics.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.hive.metastore;
+
+import static com.facebook.presto.hive.metastore.MetastoreUtil.toPartitionValues;
+import static com.google.common.base.Preconditions.checkArgument;
+import static java.util.Objects.requireNonNull;
+
+public class MetastoreApiPartitionWithStatistics
```

Comment:
Do we really need this class? This is what I'm thinking: https://github.com/highker/presto/commit/ba2800feeae0225eccf6eaa614f781fac672c717